hummingbot/client/liquidity_bounty/terms_and_conditions.txt

```diff
@@ -3,30 +3,72 @@
+
+Risk Warning
+Cryptocurrencies and digital assets are subject to high market risk. CoinAlpha is not responsible for any direct, indirect or consequential losses as a result of your participation in this Liquidity Bounty Program. Please exercise caution before deciding whether to decision to participate and transact in tokens.
+
+Legal Disclaimer
+CoinAlpha makes no claim or endorsement as to the regulatory classification of any token, including tokens paid out in this Bounty Program. Before signing up for this program, users should check with appropriate advisers and authorities to ensure that receipt, purchase and/or sale of these tokens is consistent with the rules of their relevant legal jurisdiction.
+
+Users are prohibited from utilizing Hummingbot to trade any tokens that they are prohibited from buying, selling, or trading by the rules of their local jurisdiction. By using Hummingbot and participating in any bounty program, users are acknowledging that they are allowed, consistent with any applicable laws and regulations, to use Hummingbot to buy, sell, trade, or receive as bounty any tokens they buy, sell, trade, or receive.
+
+Hummingbot is open-source software available globally and CoinAlpha makes no assurances or representations about whether a given use of the software is permitted in their local jurisdiction -- or about the regulatory classification of any tokens used or traded with Hummingbot or on Hummingbot-linked exchanges. Users are responsible for consulting with their legal advisors. By downloading Hummingbot, users acknowledge that neither CoinAlpha nor any Hummingbot developers are responsible for any legal or economic consequences stemming from their use of the trading bot or bounty program participation.
+
+Updating this Policy
+CoinAlpha reserves the right, at its sole discretion, to update, modify, and/or make any changes to the terms of this policy, including in order to improve Services or comply with any applicable law, regulations, or orders.
 
-Hummingbot Privacy Policy
-Hummingbot Apache License 2.0
-Hummingbot Website Terms of Use
-Each Token Issuer’s respective Privacy Policy that relates to your use of Hummingbot
+Upon updating this policy, CoinAlpha will give notice to its Users on this page and possibly within the Software and/or - as far as technically and legally feasible - sending a notice to Users via any contact information available to CoinAlpha. Should the changes affect processing activities performed on the basis of user consent, CoinAlpha shall collect new consent from the user, where required.
 
+It is strongly recommended to check this page often, referring to the date of the last modification listed at the bottom.
```

Comment:
Is the date of last update anywhere?   Might want to reword this and paste the link to the policy, saying to refer to that rather than this "page"